ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -404,24 +402,11 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
               '' : this.props.workspace.researchPurpose.diseaseOfFocus}
         }});
         if (this.isMode(WorkspaceEditMode.Duplicate)) {
-          this.setState({workspace: {
-            ...this.props.workspace,
-              // Replace potential nulls with empty string or empty array
-            researchPurpose: {
-              ...this.props.workspace.researchPurpose,
-              populationDetails: !this.props.workspace.researchPurpose.populationDetails ?
-                [] : this.props.workspace.researchPurpose.populationDetails,
-              diseaseOfFocus: !this.props.workspace.researchPurpose.diseaseOfFocus ?
-                '' : this.props.workspace.researchPurpose.diseaseOfFocus}
-          }});
-          if (this.isMode(WorkspaceEditMode.Duplicate)) {
-            this.setState({workspace: {
-              ...this.props.workspace,
-              // This is the only field which is not automatically handled/differentiated
-              // on the API level.
-              name: 'Duplicate of ' + this.props.workspace.name
-            }});
-          }
+          // This is the only field which is not automatically handled/differentiated
+          // on the API level.
+          this.setState(fp.set(['workspace', 'name'], 'Duplicate of ' + this.props.workspace.name));
+          // if the original workspace was reviewed, it's unlikely that we need a re-review
+          this.updateResearchPurpose('reviewRequested', false);
```

Comment:
New behavior as requested by Product: Duplicating a Yes Workspace will default to creating a No Workspace workspace as the duplicate

Reply:
I think it would be good to get a detailed spec for everything that's supposed to change or stay the same. Lots of tests and paths to update I suspect.